eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -161,6 +166,91 @@ govActionIdToText (GovernanceActionId (TxId txidHash) (GovernanceActionIx ix)) =
     <> Text.pack "#"
     <> Text.pack (show ix)
 
+data AnchorDataHash
+
+data Anchor c = Anchor
+  { anchorUrl :: !Url
+  , anchorDataHash :: !(SafeHash c AnchorDataHash)
+  }
+  deriving (Eq, Show, Generic)
+
+instance NoThunks (Anchor c)
+
+instance Crypto c => NFData (Anchor c) where
+  rnf = rwhnf
```

Comment:
I'm curious why you didn't auto-derive this.

Reply:
Few reasons.

1. It is easy to do. Just one line.
2. It is then clear that the type is alwasy in NF
3. There is no runtime overhead due to generic deriving. NFData is mostly (maybe even only) used in benchmarking, so having no overhead is beneficial, but not required. 

Because of all those reason I always prefer to defined NFData manually. I just don't always have time to do it.

Reply:
thanks for the explanation!